msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Build apps for anonymous user
+author: v-sdhakshina
+description: In this article, learn how to test the experience delivered to the anonymous users in meeting apps with all admin settings.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+---
+
+# Build apps for anonymous users
+
+You can test the experience delivered to anonymous users by selecting the meeting join url present in the meeting invite and joining the meeting via incognito window.
+
+> [!NOTE]
+>
+> * Side panel tabs and content bubbles aren't available for anonymous users at the moment.
```

Comment:
I was able to create a app with a side panel that anonymous users were able to open in R0, is this something that requires a code change? Or why does this work for me?

Reply:
@eoinobrien were you using isAnonymousAccessAllowed flag in your app? The feature to support isAnonymousAccessAllowed flag in progress and available in R0.

Reply:
we are strategically not announcing tab support with isAnonymousAccessAllowed property yet